examples/module-federation/app1/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "app1",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-cli serve",
```

```suggestion
    "start": "webpack serve",
```
I would avoid using `webpack-cli` although it does work as well.